src/decode.c

```diff
@@ -781,6 +801,22 @@ void DecodeGlobalConfig(void)
             decoder_max_layers = value;
         }
     }
+    GetPacketMaxAlertConfig();
+}
+
+void GetPacketMaxAlertConfig(void)
+{
+    intmax_t max = 0;
+    if (ConfGetInt("packet-alert-max", &max) == 1) {
+        if (max <= 0 || max > UINT8_MAX) {
+            SCLogWarning(SC_ERR_INVALID_VALUE,
+                    "Invalid value for packet-alert-max, default value set instead");
+            packet_alert_max = PACKET_ALERT_MAX;
```

Comment:
Is this necessary ?

Reply:
We thought it would be good to check if we don't have invalid values here, and Victor suggested that I followed this style found elsewhere...

Reply:
The check is necessary indeed.
The line `packet_alert_max = PACKET_ALERT_MAX;` does not seem to be.

Reply:
Ok, got it now, thanks for bearing with me xD